### PR TITLE
docs: add guide on how to create syncs 

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: #6946

### Description:

Add section on how to create Syncs

### CODE REVIEW

File: docs/docs/guides/how-to-create-syncs.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import DashboardOverviewThreeDotMenu from './assets/sync-with-google-sheets-dashboard-overview-three-dot-menu.png';
+
+# Syncs
+
+Syncs let you send your chart results to a Google Sheet in your Google Drive on a schedule. You can set up as many syncs as you like, and if you make any changes to the content, Lightdash will update the syncs the next time they’re sent.
+
+## Enabling scheduled deliveries
+
+If you are running Lightdash on self-host, make sure you have set up Google Sheets integration before you start. See [setting up Google Sheets](self-host/customize-deployment/google-sheets-integration.mdx) for more information.
+If you're not on a self-hosted instance (e.g. Cloud or Enterprise), then your instance comes preconfigured with Google Sheets integration, so you don't need to do anything to enable it.
```
Comment (hamzahc1) [inline docs/docs/guides/how-to-create-syncs.mdx:16]: There should be a line break before this sentence. 

> If you're using Lightdash Cloud, this is already set up and available for you to use immediately.

File: docs/docs/guides/how-to-create-syncs.mdx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Only users with [admin or edit access](/references/roles.mdx) can create Syncs. To create a Sync, click on the three-dot menu in the top-right corner of your chart, then click on `Sync with Google Sheets`.
+
+<img
+    src={ThreeDotMenu}
+    width="300"
+    height="276"
+    style={{ display: 'block', margin: '0 auto 20px auto' }}
+/>
+
+You'll get a modal that pops up and shows you the list of existing Syncs for your saved chart, and gives you the option to `Create new Sync`.
```
Comment (hamzahc1) [inline docs/docs/guides/how-to-create-syncs.mdx:29]: The pop-up show you a list of any existing Syncs for the saved chart, you also have the option to `Create new Sync`.

File: docs/docs/guides/how-to-create-syncs.mdx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+<img
+    src={CreateNewButtonModal}
+    width="300"
+    height="605"
+    style={{ display: 'block', margin: '0 auto 20px auto' }}
+/>
+
+If you click on `Create new Sync`, you'll be brought to a configuration screen for your new Sync.
```
Comment (hamzahc1) [inline docs/docs/guides/how-to-create-syncs.mdx:38]: Clicking `Create new Sync` takes you to the configuration screen to setup a new Google Sheets Sync.

File: docs/docs/guides/how-to-create-syncs.mdx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+<img
+    src={CreateNewModal}
+    width="300"
+    height="745"
+    style={{ display: 'block', margin: '0 auto 20px auto' }}
+/>
+
+## Sync options
+
+-   **Name**: this is the name of your Sync.
```
Comment (hamzahc1) [inline docs/docs/guides/how-to-create-syncs.mdx:49]: > This is the name of your Sync in Lightdash

File: docs/docs/guides/how-to-create-syncs.mdx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+-   **Frequency**: this is how often your Google Sheet will be synced. You can make this hourly, daily, weekly, monthly, or custom - each frequency has its own options. For example, with a monthly schedule, you set the day of the month and the time that you want your Sync to be happen. The custom frequency lets you write out your own custom Cron expression if you need something more specific than our standard options. All times for the scheduler are in UTC.
+-   **Select Google Sheet**: here, you can select the Google Sheet (from your Google Drive) that you want to sync your chart results to. Note: you have to have created the Google Sheet before you can select it here.
```
Comment (hamzahc1) [inline docs/docs/guides/how-to-create-syncs.mdx:51]: > Use the file picker to select the Google Sheet that you'd like to sync your chart results to. Note: You need to sync to an existing Google Sheet, so ensure you have created it before you can select it here.

**Review by hamzahc1 — CHANGES_REQUESTED** [review]
One section of the docs we could add @joaoviana is to say that the Sync will also create a metadata tab in the selected Google Sheet that will be updated with each sync (This could be  at the end of the current "Creating a Sync" section:

> When you create a Sync, Lightdash also creates a special tab in your Google Sheet called `metadata` which includes information on when the sheet was last updated and the frequency interval. This allows users to easily understand how often data is refreshed inside their Google Sheet.

File: docs/docs/guides/how-to-create-syncs.mdx [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+import DashboardOverview from './assets/sync-with-google-sheets-dashboard-overview.png';
+import DashboardOverviewThreeDotMenu from './assets/sync-with-google-sheets-dashboard-overview-three-dot-menu.png';
+
+# Syncs
+
+Syncs let you send your chart results to a Google Sheet in your Google Drive on a schedule. You can set up as many syncs as you like, and if you make any changes to the content, Lightdash will update the syncs the next time they’re sent.
+
+## Enabling scheduled deliveries
+
+If you are running Lightdash on self-host, make sure you have set up Google Sheets integration before you start. See [setting up Google Sheets](self-host/customize-deployment/google-sheets-integration.mdx) for more information.
```
[the remark this answers is not in the document]
Comment (joaoviana) [reply docs/docs/guides/how-to-create-syncs.mdx:15]: done

File: docs/docs/guides/how-to-create-syncs.mdx (hunk 1 above)
Comment (joaoviana) [reply docs/docs/guides/how-to-create-syncs.mdx:16]: done

File: docs/docs/guides/how-to-create-syncs.mdx [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+If you're not on a self-hosted instance (e.g. Cloud or Enterprise), then your instance comes preconfigured with Google Sheets integration, so you don't need to do anything to enable it.
+
+## Creating a Sync
+
+Only users with [admin or edit access](/references/roles.mdx) can create Syncs. To create a Sync, click on the three-dot menu in the top-right corner of your chart, then click on `Sync with Google Sheets`.
```
[the remark this answers is not in the document]
Comment (joaoviana) [reply docs/docs/guides/how-to-create-syncs.mdx]: done

File: docs/docs/guides/how-to-create-syncs.mdx (hunk 3 above)
Comment (joaoviana) [reply docs/docs/guides/how-to-create-syncs.mdx]: done

File: docs/docs/guides/how-to-create-syncs.mdx (hunk 4 above)
Comment (joaoviana) [reply docs/docs/guides/how-to-create-syncs.mdx]: done

File: docs/docs/guides/how-to-create-syncs.mdx (hunk 5 above)
Comment (joaoviana) [reply docs/docs/guides/how-to-create-syncs.mdx]: done

**Review by hamzahc1 — APPROVED** [review]
Looks good!

File: docs/docs/guides/how-to-create-syncs.mdx [hunk 8]
```diff
@@ -47,6 +47,8 @@ Clicking `Create new Sync` takes you to the configuration screen to setup a new
 
 When you create a Sync, Lightdash also creates a special tab in your Google Sheet called metadata which includes information on when the sheet was last updated and the frequency interval. This allows users to easily understand how often data is refreshed inside their Google Sheet.
 
+> Note: If you are uploading data from a chart, the data will be uploaded to the first tab in your selected Google Spreadsheet and it will overwrite all the content.
+
```
Comment (joaoviana) [inline docs/docs/guides/how-to-create-syncs.mdx:51]: @hamzahc1 can you have a quick look at this?

**Pull request merged**